MediaBrowser.Common/Extensions/CollectionExtensions.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.Generic;
+
+namespace MediaBrowser.Common.Extensions
+{
+    // The MS CollectionExtensions are only available in netcoreapp
+    public static class CollectionExtensions
+    {
+        public static TValue GetValueOrDefault<TKey, TValue> (this IReadOnlyDictionary<TKey, TValue> dictionary, TKey key)
```

Comment:
Doesn't this mirror the `FirstOrDefault` type behaviour from LINQ?

Reply:
Except it's much faster (I'm guessing) and already a convenience function they've added to netcoreapp.